lib/ZwaveClient.js

```diff
@@ -522,23 +520,22 @@ function sceneEvent (nodeid, sceneCode) {
 
 function notification (nodeid, notif, help) {
   // eslint-disable-next-line no-unused-vars
-  var msg
   var ozwnode = this.nodes[nodeid]
   switch (notif) {
     case 0:
-      msg = 'node' + nodeid + ': message complete'
+    // msg = 'node' + nodeid + ': message complete'
       break
     case 1:
-      msg = 'node' + nodeid + ': timeout'
+    // msg = 'node' + nodeid + ': timeout'
       break
     case 2:
```

Comment:
this could probably be tidied up a bit?
like:
```js
case 0:
case 1:
case 2:
  break
```

Reply:
sure